src/main/resources/citizen/templates/document/claimantDirectionsQuestionnaire.html

```diff
@@ -383,38 +426,43 @@ <h2 class="heading-medium">
     </td>
     <td>
       <div>
-        {{ hearingContent.witness.selfWitness }}
+        {{ hearingContent.witness.selfWitness | capitalize }}
       </div>
     </td>
 
   </tr>
-  <tr>
-    <td>
-      <div>
-        Other witnesses
-      </div>
-    </td>
-    <td>
-      <div>
-        {{ hearingContent.witness.noOfOtherWitness }}
-      </div>
-    </td>
 
-  </tr>
+  {% if hearingContent.witness.noOfOtherWitness is not empty %}
+    <tr>
+      <td>
+        <div>
+          Other witnesses
+        </div>
+      </td>
+      <td>
+        <div>
+          {{ hearingContent.witness.noOfOtherWitness }}
+        </div>
+      </td>
 
-  <tr>
-    <td>
-      <div>
-        Dates unavailable
-      </div>
-    </td>
-    <td>
-      <div>
-        {% for unavailableDate in hearingContent.unavailableDates %}
-        {{ unavailableDate }} <br/>
-        {% endfor %}
-      </div>
-    </td>
+    </tr>
+  {% endif %}
+
+  {% if hearingContent.unavailableDates is not empty %}
```

Comment:
if this is empty it should say None. I remember this from the call with marc

Reply:
done